lib/util/noop.js

```diff
@@ -1,6 +1,6 @@
 "use strict";
 
-var noop = function() {};
+var noop = Function.prototype;
```

Comment:
This is currently how noop is implemented in lodash https://github.com/lodash/lodash/issues/846#issuecomment-68343994